### DISCUSSION TITLE
使用编译出来的test程序显示的不对

### DISCUSSION THREAD

**Discussion by taynpg** [sxwnl/sxwnl-cpp#13] [General]
![image](https://github.com/sxwnl/sxwnl-cpp/assets/59216127/a25a2378-2956-4a44-bd43-07afe0195636)

这个怎么显示的是未知月？

**Comment by qaiu** [comment]
稍等，我查一下

**Comment by qaiu** [comment]
确实是个bug，稍后修复

**Reply by taynpg** [reply]
好的

**Comment by taynpg** [comment]
其实我还有个疑问，大佬，顺便在这里问一下，就是源码中为什么改成了三方的STL而不是使用STL呢?

**Reply by qaiu** [reply]
考虑到STL太重了实际上很多容器和算法并没有用到，所以找了一个三方实现的轻量STL

**Reply by qaiu** [reply]
这样编译出的二进制体积可以减少好多倍

**Reply by qaiu** [reply]
其中包括string在内的也用了第三方实现

**Reply by taynpg** [reply]
OK，懂了，感谢！

**Comment by qaiu** [comment]
已修复\(^o^)/~
![8f4b0192752c1680a999b4d43bf7198](https://github.com/sxwnl/sxwnl-cpp/assets/29825328/0c043da4-5b99-4655-8a83-0b63806b9922)

**Reply by taynpg** [reply]
赞！
